Examples/Makefile.in

```diff
@@ -1350,6 +1350,72 @@ r_clean:
 	rm -f *.@OBJEXT@ *@SO@ NAMESPACE
 	rm -f $(RRSRC) $(RUNME).Rout .RData
 
+##################################################################
+#####                           C                           ######
+##################################################################
+
+# ----------------------------------------------------------------
+# Build a C dynamically loadable module
+# ----------------------------------------------------------------
+
+CLIBPREFIX = lib
+C_LDSHARED = @C_LDSHARED@
+CXX_LDSHARED = @CXX_LDSHARED@
+C_SO = @C_SO@
+
+c: $(SRCDIR_SRCS)
+	$(SWIG) -c $(SWIGOPT) -o $(ISRCS) $(INTERFACEPATH)
+	$(CC) -c $(CCSHARED) -I$(SRCDIR) $(CFLAGS) $(ISRCS) $(SRCDIR_SRCS) $(INCLUDES)
+	$(C_LDSHARED) $(CFLAGS) $(OBJS) $(IOBJS) $(LIBS) -o $(CLIBPREFIX)$(TARGET)$(C_SO)
+
+c_cpp: $(SRCDIR_SRCS)
+	$(SWIG) -c++ -c $(SWIGOPT) -o $(ICXXSRCS) $(INTERFACEPATH)
+	$(CXX) -c $(CCSHARED) -I$(SRCDIR) $(CXXFLAGS) $(ICXXSRCS) $(SRCDIR_CXXSRCS) $(INCLUDES)
+	$(CXX_LDSHARED) $(CFLAGS) $(OBJS) $(IOBJS) $(LIBS) $(CPP_DLLIBS) -o $(CLIBPREFIX)$(TARGET)$(C_SO)
```

Comment:
I would suggest prefixing this and the `C_LDSHARED` command above with `$(COMPILETOOL)`, which seems to be consistent with how that's used with other target languages.

Then the change in `Examples/test-suite/common.mk` to override `C_LDSHARED=true` etc wouldn't be necessary.

Reply:
Thanks!